### PR TITLE
Add twitter URLs for speakers

### PR DESCRIPTION
**Pull request by joshmoore**
Add twitter information for speakers

### CODE REVIEW

**Comment by sbesson** [comment]
I assume the submodule bump was not done on purpose

File: _data/ome2021.json [hunk 1]
```diff
@@ -189,7 +189,8 @@
             "@type": "Organization",
             "name": "Micelio"
           },
-          "sameAs": "https://www.micelio.be/people.html"
+          "sameAs": "https://www.micelio.be/people.html",
+          "sameAs": "https://twitter.com/andrawaag"
```
Comment (sbesson) [inline _data/ome2021.json:193]: This is flagged as duplicate unique property and will need to be reformatted as

```
          "sameAs": [
            "https://www.micelio.be/people.html",
            "https://twitter.com/andrawaag"
          ]
```

Also https://github.com/ome/www.openmicroscopy.org/blob/8ac24418ef3c8f2eee7fcd139f82e9457b8017ed/_includes/ome2021_program_agenda.html#L24 will need to be updated to loop over all values and filter the Twitter URLs.

**Comment by joshmoore** [comment]
Pushed an attempt but it looks like `sameAs` is coming through as a single string. Is this a problem you've already seen elsewhere?

(In general, this pattern of stripping out addresses which match X isn't going to scale well)

**Comment by sbesson** [comment]
> (In general, this pattern of stripping out addresses which match X isn't going to scale well)

Agreed. I have seen the following pattern used and recommended in a few places:

- `{Person/Organization}.url` == official URL
- `{Person/Organization}.sameAs` == list of social media URLs

In this case, this would remove the need for conditional handling. But the question would still arise if we wanted to expose multiple sameAs properties (e.g. LinkedIn + Twitter + GitHub),

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by joshmoore** [comment]
Guessing this can be closed now.

**Pull request closed without merging**